packages/web-components/src/components/tabs-extended/__stories__/tabs-extended.stories.ts

```diff
@@ -71,6 +71,323 @@ export const Default = ({ parameters }) => {
   `;
 };
 
+export const WithCTABlockItemRows = ({ parameters }) => {
```

Comment:
@proeung glad we got the overlapping issue worked out and thank you for the explanation! I have mixed feelings about this, for now I would like to keep it to the CodeSandbox so we aren't blocking AEM and we can get the fix in. 

Before we add to storybook I'd like to update: 
1. **The story name** – "Content block" does not fit because this isn't using the content block heading (expressive-heading-05). If anything "with content items" is more applicable. 
2. **The content** – this is not a good example of how design content or use a `content item`
   - Each item should have a heading
   - The body copy is more of a category or label, if it's body copy it should be descriptive
   - The CTAs are too long

Do you have any idea which designer I can speak to about this component/pattern?

Reply:
I've removed the Storybook example for now, and I'll open up a ticket for making sure we have automated test coverage for this scenario.